test/test_sparse_csr.py

```diff
@@ -154,6 +159,7 @@ def test_dense_convert(self, device, dtype):
     @coalescedonoff
     @onlyCPU
     @dtypes(torch.double)
+    @unittest.skipIf(IS_MACOS or IS_WINDOWS, "MKL doesn't work on windows or mac")
```

Comment:
This skip test is not needed here. Because c++ code check if there is not MKL or if there is windows. In any other case there is specific code to handle that. Maybe is failing because there was a bug which is solved here. https://github.com/pytorch/pytorch/pull/58622

Reply:
mentioned here too: 
https://github.com/pytorch/pytorch/issues/58757

Reply:
sounds good. although I do prefer skipping here and comment with a TODO with the issue number attached. once #58757 ready for review we can un-skip.

Reply:
Hmm, MKL is pretty much platform agnostic. Why wouldn't it work on Windows or Mac?